src/core/src/core_logic/PatchInstaller.py

```diff
@@ -192,7 +228,7 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
                 break
 
             # point in time status
-            progress_status = self.progress_template.format(str(datetime.timedelta(minutes=remaining_time)), str(attempted_parent_update_count), str(successful_parent_update_count), str(failed_parent_update_count), str(installed_update_count - successful_parent_update_count),
+            progress_status = self.progress_template.format(str(datetime.timedelta(minutes=remaining_time)), str(self.attempted_parent_package_install_count), str(self.successful_parent_package_install_count), str(self.failed_parent_package_install_count), str(installed_update_count - self.successful_parent_package_install_count),
                                                             "Processing package: " + str(package) + " (" + str(version) + ")")
```

Comment:
Similar to other comment/s raised earlier, extension state check, maintenance window and point in time status should happen before we start processing both in single and batched patches

Reply:
> > **Code changes done**: Product and test code changes are completed, only review is pending.
> > **Manual testings completed**: (a) Tested that the installation of all the patches is successful.
> > (b) Tested with total number of available patches to install is greater than batch size (patches will be installed in multiple batches) and total number of patches to install is less than batch size (patches will be installed in single batch).
> > (c) The portal shows correct status of the patches.
> > (d) Test with different classification and packages to include and exclude. Check if status of packages is shown correctly in the portal.
> > (e) If remaining time is not available in maintenance window to install updates in batch, then do not proceed with installing the remaining batches. The remaining packages should be attempted to install sequentially. If there is not enough time available to install packages sequentially then the installation process is stopped at that time. Also, check if status is marked correctly for non-attempted packages.
> > (f) Tested install updates job for all the three package managers: Apt, Yum and Zypper.
> > (g) Verified that after batch patching is done, the get available updates is returning the list of updates available and they are getting installed as part of sequential patching.
> 
> Please add a brief description of what the PR is trying to achieve and how

Added brief description.

Reply:
Keeping this open since the conversation about this process in batched processing, is still open